capsules/src/screen.rs

```diff
@@ -383,16 +284,12 @@ impl<'a> Screen<'a> {
     }
 
     fn schedule_callback(&self, data1: usize, data2: usize, data3: usize) {
-        if !self.screen_ready.get() {
-            self.screen_ready.set(true);
```

Comment:
What was the point of this? This callback would have not been forwarded anywhere. I don't get it, and don't want to break this by accident.

Reply:
Whenever the underlying driver returns a `command_complete` the screen driver might have to execute a queued command. This is done by the `run_next_command` function. This schedules a callback for the previous action and then runs the next one.

The lines that you are asking about are the way in which `schedule_callback` differentiates between the `screen_is_ready` and the `command_complete` notifications from the underlying driver.